src/utils/strings.ts

```diff
@@ -29,32 +29,44 @@ export const checkHexStringValidity = (stringToTest: string) => {
   return /^[a-fA-F0-9]+$/.test(stringToTest)
 }
 
-export const thousandsSeparation = (input: number): string =>
-  input
-    .toString()
-    .split('')
-    .reduce(
-      (str, char, index, list) => (index > 0 && index < list.length - 1 && index % 2 === 0 ? '’' : '') + char + str,
-      ''
-    )
-
-type Suffices = [number, string][]
-
-export const SUFFICES_QUANTITY: Suffices = [
-  [1000000000, 'B'],
-  [1000000, 'M'],
-  [1000, 'K']
-]
-
-export const SUFFICES_TIME: Suffices = [
-  [1000 * 60 * 60, 'h'],
-  [1000 * 60, 'm'],
-  [1000, 's']
-]
-
-export const abbreviateValue = (n: number, suffices: Suffices, precision = 3): string =>
-  suffices.reduce((cur: string | undefined, [magnitude, suffix]) => {
-    if (cur !== undefined) return cur.toString()
-    if (n / magnitude < 1.0) return cur
-    return (n / magnitude).toPrecision(precision) + suffix
-  }, undefined) || n.toString()
+export const formatTimeForDisplay = (n: number): string => {
+  const timeSuffices: [number, string][] = [
+    [1000 * 60 * 60, 'h'],
+    [1000 * 60, 'min'],
+    [1000, 'sec']
+  ]
+
+  return (
+    timeSuffices.reduce((cur: string | undefined, [magnitude, suffix]) => {
+      if (cur !== undefined) return cur.toString()
+      if (n / magnitude < 1.0) return cur
+      return (n / magnitude).toPrecision(3) + suffix
+    }, undefined) || n.toString()
+  )
+}
+
+export const formatNumberForDisplay = (
```

Comment:
Nit: `abbreviateValue` could have been adjusted to return `[string, string, string]` and additional suffix could have been added instead (it's a general function which will work with any measurement)